--- gen/wstServer.c	2007-05-30 13:31:54.000000000 +0200
+++ gen/wstServer.c.fixup	2007-05-30 13:32:36.000000000 +0200
@@ -74,7 +74,7 @@
 SOAP_FMAC5 int SOAP_FMAC6 wst_serve_request(struct soap *soap)
 {
 	soap_peek_element(soap);
-	if ((!soap->action && !soap_match_tag(soap, soap->tag, "")) || (soap->action && !strcmp(soap->action, "http://schemas.xmlsoap.org/ws/2004/09/transfer/Get")))
+	if (soap->action && !strcmp(soap->action, "http://schemas.xmlsoap.org/ws/2004/09/transfer/Get"))
 		return soap_serve___wst__Get(soap);
 	return soap->error = SOAP_NO_METHOD;
 }
